package.json

```diff
@@ -27,11 +27,11 @@
     ],
     "license": "MIT",
     "dependencies": {
+        "ddq": "2.0.0",
```

Comment:
This creates a circular dependency.  You should list this under `peerDependencies`.  https://nodejs.org/en/blog/npm/peer-dependencies/